src/mococrw/signature.h

```diff
@@ -34,136 +35,36 @@ class SignatureUtils
     SignatureUtils() = delete;
```

Comment:
This is an all-static member class once again.

Maybe we could attach this to the CryptoContext and not have Encrypt or Decrypt Context but a PubkeyContext and Privkey Context?